> pandas/tests/computation/test_eval.py:834: error: Need type annotation for "exclude_bool" (hint: "exclude_bool: List[<type>] = ...")  [var-annotated]
pandas/tests/computation/test_eval.py:835: error: Need type annotation for "exclude_cmp" (hint: "exclude_cmp: List[<type>] = ...")  [var-annotated]

--- a/pandas/tests/computation/test_eval.py
+++ b/pandas/tests/computation/test_eval.py
@@ -831,8 +831,8 @@ class TestEvalPythonPython(TestEvalNumexprPython):
 class TestEvalPythonPandas(TestEvalPythonPython):
     engine = "python"
     parser = "pandas"
-    exclude_bool = []
-    exclude_cmp = []
+    exclude_bool: list[str] = []
+    exclude_cmp: list[str] = []
 
     def check_chained_cmp_op(self, lhs, cmp1, mid, cmp2, rhs):
         TestEvalNumexprPandas.check_chained_cmp_op(self, lhs, cmp1, mid, cmp2, rhs)

